packages/morph/src/styles/editor.scss

```
@@ -1,76 +1,82 @@
 $line-height: 1.3;
-$blockquote-border-color: #cccccc;
-$blockquote-text-color: #555555;
+$gray-light: #e5e5e5;
+$gray-medium: #cccccc;
+
+$base-size: 1rem;
+$scale-ratio: 1.2;
+
+$h6-size: $base-size;
+$h5-size: $base-size * $scale-ratio;                   
+$h4-size: $base-size * $scale-ratio * $scale-ratio;
+$h3-size: $base-size * pow($scale-ratio, 3);
+$h2-size: $base-size * pow($scale-ratio, 4);
+$h1-size: $base-size * pow($scale-ratio, 5);
+
+html {
+  font-size: 14px;
+  line-height: $line-height;
+}
+
+@mixin reset-text {
+  margin: 0;
+  padding: 0;
+}
 
 .cm-editor {
-  .cm-scroller {
-    .cm-content {
-      .cm-line {
-        line-height: $line-height;
-        margin: 0;
-        padding: 0;
-        white-space: pre-wrap;
-        position: relative;
-      }
-    }
+  .cm-scroller .cm-content .cm-line {
```

```suggestion
  & > .cm-scroller > .cm-content > .cm-line {
```

Would the following be the same?

```scss
.cm-line { ...} 
```

---

Yeah still the same, I changed it to what you wanted